air/src/proof/mod.rs

```diff
@@ -212,13 +212,30 @@ impl Deserializable for Proof {
 // HELPER FUNCTIONS
 // ================================================================================================
 
+pub struct ConjecturedSecurityBits(pub u32);
+
+pub struct ProvenSecurityBits {
+    unique_decoding: u32,
+    list_decoding: u32,
+}
+
+impl ProvenSecurityBits {
+    pub fn list_decoding(&self) -> u32 {
+        self.list_decoding
+    }
+
+    pub fn unique_decoding(&self) -> u32 {
+        self.unique_decoding
+    }
+}
+
 /// Computes conjectured security level for the specified proof parameters.
 fn get_conjectured_security(
     options: &ProofOptions,
     base_field_bits: u32,
     trace_domain_size: usize,
     collision_resistance: u32,
-) -> u32 {
+) -> ConjecturedSecurityBits {
     // compute max security we can get for a given field size
     let field_size = base_field_bits * options.field_extension().degree();
     let field_security = field_size - (trace_domain_size * options.blowup_factor()).ilog2();
```

Comment:
As far as I can tell, Conjecture 1 in ethSTARK does not include the trace domain size? Might be good to add a reference to where that comes from.

Reply:
Indeed, this is a sort of hybrid between the conjecture on the toy problem (or more succinctly toy conjecture) and  conjecture 8.4 in the proximity gaps paper.
As you suggest, it might make more sense to remove the domain size from the above so as to match the toy conjecture exactly and then add an additional "security mode" to capture Conjecture 8.4 with the exponents therein equal to 1.